tests/testthat/test-convGDX2mif.R

```diff
@@ -16,48 +16,18 @@ test_that("Test if REMIND reporting is produced as it should and check data inte
   # add GDXs for comparison here:
   gdxPaths <- NULL
 
-  if (length(gdxPaths) == 0) {
-    defaultGdxPath <- file.path(tempdir(), "fulldata.gdx")
-    if (!file.exists(defaultGdxPath)) {
-      utils::download.file("https://rse.pik-potsdam.de/data/example/remind2_test-convGDX2MIF_fulldata.gdx",
-        defaultGdxPath,
-        mode = "wb", quiet = TRUE
-      )
-    }
-    gdxPaths <- defaultGdxPath
-  }
-
-  # finds for each AMT scenario the most recent, successfully converged GDX,
-  # that is no older than 30 days
-  .findAMTgdx <- function(gdxPaths = NULL, scenario = NULL) {
-    # regex for "%Y-%m-%d_%H.%M.%S" timestamp
-    datetimepattern <- "[0-9]{4}-[0-9]{2}-[0-9]{2}_[0-9]{2}\\.[0-9]{2}\\.[0-9]{2}"
-
-    .did_REMIND_finish <- function(path) {
-      logpath <- sub("fulldata.gdx$", "full.log", path)
-      return(   file.exists(logpath)
-             && "*** Status: Normal completion" %in% readLines(logpath,
-                                                               warn = FALSE))
-    }
-
-    .latest_run_of_scenario <- function(path) {
-      # sort by scenario and decreasing time, latest runs come first
-      path <- path %>%
-        sort(decreasing = TRUE)
-
-      # duplicates are older than the preceding runs
-      dup <- duplicated(sub(paste0("_", datetimepattern), "",
-                            basename(dirname(path))))
+  gdxList <- c("fulldata-release.gdx" = "https://rse.pik-potsdam.de/data/example/remind2_test-convGDX2MIF_fulldata.gdx",
+               "fulldata-AMT.gdx"     = "https://rse.pik-potsdam.de/data/example/remind2_test-convGDX2MIF_SSP2EU-NPi-AMT.gdx")
 
-      # everything that is no duplicate is the latest run of that scenario
-      return(path[!dup])
+  if (length(gdxPaths) == 0) {
+    for (i in seq_along(gdxList)) {
+      from <- gdxList[i]
+      to   <- file.path(tempdir(), names(gdxList[i]))
+      if (!file.exists(to)) {
```

Comment:
Not sure whether that makes sense. Because once you have a gdx file there, you will never download the new AMT file, right? Can you maybe check with @0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q on how you can integrate the caching he suggested [here](https://github.com/pik-piam/remind2/pull/478/files) (but somehow never merged)?

Reply:
Interesting. Isn't the tempdir() different every time it is created?

Reply:
Ah, yes, you are right for course.

Reply:
Which means the gdx files will be downloaded again and again, which Michaja wanted to avoid if I unterstand correctly. I'll merge anyway and we can still integrate Michaja's solution.